builtin/tag.c

```diff
@@ -106,7 +106,28 @@ static int delete_tag(const char *name, const char *ref,
 static int verify_tag(const char *name, const char *ref,
 				const unsigned char *sha1)
 {
-	return gpg_verify_tag(sha1, name, GPG_VERIFY_VERBOSE);
+	int flags;
+	flags = GPG_VERIFY_VERBOSE;
+
+	if (fmt_pretty) {
+		//verify_ref_format(fmt_pretty);
+
```

Comment:
I am reluctant to verify the format string here because it needs to be done only once whereas this function is called for every tag to verify. Maybe [here](https://github.com/lukpueh/git/blob/010fc57206b74febc2b4bc9ad21510e26950b406/builtin/tag.c#L449-L450) would be a better place.